server.js

```diff
@@ -16,6 +16,8 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var app = express();
 
+var COMMENTS_FILE = path.join(__dirname, 'comments.json');
```

Comment:
Matches the script-relative path pattern used to reference the project's 'public' directory on [line 23](https://github.com/reactjs/react-tutorial/pull/84/files#diff-78c12f5adc1848d13b1c6f07055d996eR23).